src/plugins/kernel_install/gentoo.sh

```diff
@@ -0,0 +1,48 @@
+# Gentoo package names
```

Comment:
This change does not belong to this commit, fwiu, in this commit, you are focusing on the Gentoo basic setup, meaning you need to move all changes in the `src` folder to its specific commit. In this commit, you should only care about the setup.

Reply:
Okay I'll re base accordingly